packages/react/src/components/List/List.tsx

```diff
@@ -228,7 +228,7 @@ export class List<T = any> extends React.Component<IListProps<T>, IListState<T>>
       itemsPerPage = pageSpecification.itemCount;
 
       const requestedIndexIsInPage = itemIndex <= index && itemIndex + itemsPerPage > index;
-      if (requestedIndexIsInPage) {
+      if (requestedIndexIsInPage && this._scrollElement) {
         // We have found the page. If the user provided a way to measure an individual item, we will try to scroll in
         // just the given item, otherwise we'll only bring the page into view
         if (measureItem && this._scrollElement) {
```

Comment:
By testing `this._scrollElement` in the parent `if` we can avoid duplication to avoid confusion on this one, right?